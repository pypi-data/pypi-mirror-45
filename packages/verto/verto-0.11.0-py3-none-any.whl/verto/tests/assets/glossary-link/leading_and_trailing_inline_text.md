We've already had a taste of {glossary-link term="Regular expression"}regular expressions{glossary-link end} in the [getting started](chapters/formal-languages.html#getting-started) section. They are just a simple way to search for things in the input, or to specify what kind of input will be accepted as legitimate.
For example, many web scripting programs use them to check input for patterns like dates, email addresses and URLs. They've become so popular that they're now built into most programming languages.