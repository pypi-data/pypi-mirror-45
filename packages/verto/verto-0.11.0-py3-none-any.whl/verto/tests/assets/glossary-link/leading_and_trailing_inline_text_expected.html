<p>We've already had a taste of <a class="glossary-term" data-glossary-term="Regular expression">regular expressions</a> in the <a href="chapters/formal-languages.html#getting-started">getting started</a> section. They are just a simple way to search for things in the input, or to specify what kind of input will be accepted as legitimate.
For example, many web scripting programs use them to check input for patterns like dates, email addresses and URLs. They've become so popular that they're now built into most programming languages.</p>
